src/couch_mrview/src/couch_mrview.erl

```diff
@@ -708,5 +708,5 @@ make_view_changes_opts(StartSeq, Options, Args) ->
         true ->
             couch_mrview_util:changes_key_opts(StartSeq, Args);
         false ->
-            [[{start_key, {StartSeq+1, <<>>}}] ++ Options]
+            [[{start_key, {StartSeq, <<>>}}] ++ Options]
```

Comment:
Yes, this makes sense. This is really subtle bug. Need to take care of different situations. We can use a different PR to address this.